src/generator/templates/package.json.njk

```diff
@@ -23,14 +25,19 @@
     "lint": "gts check",
     "compile": "tsc -p .",
     "prepare": "npm run compile",
-    "docs": "typedoc --out docs/"
+    "docs": "typedoc --out docs/",
+    "webpack": "npx webpack"
```

Comment:
You don't need to use `npx` here since it's installed as a dev dependency and run in the context of an npm script.